DrawerKitDemo/DrawerKitDemo/PresentedViewController.swift

```diff
@@ -11,16 +11,26 @@ class PresentedViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.notificationToken = NotificationCenter.default
-            .addObserver(name: DrawerNotification.drawerExteriorTappedNotification) {
-            (notification: DrawerNotification, object: Any?) in
-            switch notification {
-            case .drawerExteriorTapped:
-                print("drawerExteriorTapped")
-            default:
-                break
```

Comment:
### Note

Useless pattern-matching in this example code is highly discouraged, and this PR will fix this issue.